topologypolicy/parse.go

```
@@ -0,0 +1,121 @@
+
+const maxJSONDepth = 32
+
+func Load(path string) (Contract, error) {
+	file, err := os.Open(path)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return Contract{}, fmt.Errorf("open topology contract: file does not exist")
+		}
+		return Contract{}, fmt.Errorf("open topology contract: file is unavailable")
+	}
+	defer file.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `file.Close` is not checked

(errcheck)